documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager.Shared/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Do we need have a package.json file for the shared library?

Reply:
Removed.